README.md

```diff
@@ -156,7 +156,7 @@ Now every object on the map can be hovered (however, you can still use css hover
 [docs with webpackbin examples](./DOC.md) (In progress)
 
 * local develop example (new)
-[develop example](./develop)
+[develop example](../../tree/develop)
```

Comment:
I think it needs to be changed to /example right?

Reply:
Yes, sorry. I got confused between develop and example. I thought its the develop branch the doc is referring to.